proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/route/RouteActivity.java

```diff
@@ -162,12 +161,11 @@ protected Permission convertToPermission(int perm) {
     }
 
     protected List<org.apache.rocketmq.proxy.common.Address> convertToAddressList(Endpoints endpoints) {
-        int port = ConfigurationManager.getProxyConfig().getGrpcServerPort();
         List<org.apache.rocketmq.proxy.common.Address> addressList = new ArrayList<>();
         for (Address address : endpoints.getAddressesList()) {
             addressList.add(new org.apache.rocketmq.proxy.common.Address(
                 org.apache.rocketmq.proxy.common.Address.AddressScheme.valueOf(endpoints.getScheme().name()),
-                HostAndPort.fromParts(address.getHost(), port))
+                HostAndPort.fromParts(address.getHost(), address.getPort()))
```

Comment:
Add a configuration to decide whether to use the configured port in proxyConfig or the port in the endpoint is better

Reply:
yes

Reply:
I have added a configuration to support the k8s NodePort mode according to your suggestion, and there is another Tests by Bazel Action run fail, which I do not understand